### PR TITLE
Release cluster-operator 0.23.9 patch version for aws installation

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#10211

Creating new patch release which would include cluster-operator 0.23.9. 

- 9.0.5
- 9.3.1

### CODE REVIEW

**Comment by tomahawk28** [comment]
Tested on `ginger`

File: aws.yaml [hunk 1]
```diff
@@ -301,9 +301,61 @@ kind: Release
...
+      version: 1.2.0
+  components:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.7.1
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      version: 0.23.9
```
Comment (tomahawk28) [inline aws.yaml:341]: Only this part is different compared to the previous release.

File: aws.yaml [hunk 2]
```diff
@@ -301,9 +301,61 @@ kind: Release
...
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.3.0
+  name: v9.3.1
 spec:
   state: active
+  date: 2020-05-12T11:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.1
```
Comment (rossf7) [inline aws.yaml:310]: Can we use 1.2.2? This has the Helm 3 change to use Release.Revision so we are prepared.

https://github.com/giantswarm/cert-exporter/tree/v1.2.2

File: aws.yaml [hunk 3]
```diff
@@ -301,9 +301,61 @@ kind: Release
...
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.0
```
Comment (rossf7) [inline aws.yaml:326]:
```suggestion
      version: 1.7.1
```

Same thing for net-exporter but there are some other changes there. So I'm fine if we hold that back.

https://github.com/giantswarm/net-exporter/releases/tag/v1.7.1

**Comment by tomahawk28** [comment]
@rossf7 I added releases note in here, release-notes/aws/v9.3.1.md

File: aws.yaml (hunk 2 above)
Comment (tomahawk28) [reply aws.yaml:310]: If its only helm3, I think it makes sense include in here.

File: release-notes/aws/v9.0.5.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# :zap: Giant Swarm Release v9.0.5 for AWS :zap:
+
+This release fixes a problem in user values migration logic for apps, and fixing release templates to support Helm 3.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.0.5.md:3]:
```suggestion
This release fixes a bug causing the NGINX IC app to be removed from a cluster.

It also modifies release templates to support the coming upgrade to Helm 3.
```

Comment (cokiengchiara) [reply release-notes/aws/v9.0.5.md:3]: I wanted to make the value of the bug fix clearer. Also to communicate the coming Helm 3 upgrade. WDYT? @tomahawk28 @rossf7 ?

Same applies to 9.3.1

File: release-notes/aws/v9.3.1.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# :zap: Giant Swarm Release v9.3.1 for AWS :zap:
+
+This release fixes a problem in user values migration logic for apps, and fixing release templates to support Helm 3.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.3.1.md:3]:
```suggestion
This release fixes a bug causing the NGINX IC app to be removed from a cluster.

It also modifies release templates to support the coming upgrade to Helm 3.
```

File: release-notes/aws/v9.0.5.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# :zap: Giant Swarm Release v9.0.5 for AWS :zap:
+
+This release fixes a bug causing the NGINX IC app to be removed from a cluster.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.0.5.md:3]:
```suggestion
This release fixes a rare bug that would prevent the NGINX IC from being installed on a new cluster. 

This bug would only occur on cluster creation if you had a nginx-ingress-controller-user-values configmap in the kube-system namespace while the cluster was still initialising.

Solution Engineers have already done the manual fix for affected customers.
```

File: release-notes/aws/v9.3.1.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# :zap: Giant Swarm Release v9.3.1 for AWS :zap:
+
+This release fixes a bug causing the NGINX IC app to be removed from a cluster.
```
Comment (cokiengchiara) [inline release-notes/aws/v9.3.1.md:3]:
```suggestion
This release fixes a rare bug that would prevent the NGINX IC from being installed on a new cluster. 

This bug would only occur on cluster creation if you had a nginx-ingress-controller-user-values configmap in the kube-system namespace while the cluster was still initialising.

Solution Engineers have already done the manual fix for affected customers.
```

**Comment by tomahawk28** [comment]
Is it safe with one approval? perhaps one more from @giantswarm/team-batman ?

**Pull request merged**